R/rd-html.R

```diff
@@ -434,15 +434,9 @@ as_html.tag_code <-         function(x, ..., auto_link = TRUE) {
 
 #' @export
 as_html.tag_preformatted <- function(x, ...) {
-  text <- flatten_text(x, ...)
-
-  # Need to unescape so that highlight_text() can tell if it's R code
-  # or not. It'll re-escape if needed
-  text <- unescape_html(text)
-  highlight_text(text)
+  sprintf("<pre><code>%s</code></pre>", flatten_text(x, ...))
```

Comment:
Can you add a comment mentioning that highlighting is now done in `tweak_reference_topic_html()`?